rest-server/src/routes/indexV2.js

```diff
@@ -28,7 +28,7 @@ router.route('/').all((req, res) => {
 });
 
 router.use('/template', templateRouter);
-router.use('/job', jobV2Router);
+router.use('/jobs', jobV2Router);
```

Comment:
Should we unify the API url prefix to singular form starting from the "JobV2"? 
![image](https://user-images.githubusercontent.com/17514649/44017596-76444494-9f0b-11e8-9234-5cf90660225d.png)